src/repositories/mod.rs

```diff
@@ -833,104 +833,93 @@ impl RepoEditOptions {
     pub fn new<N>(name: N) -> Self
     where
         N: Into<String>,
     {
-        RepoEditOptionsBuilder {
-            name: name.into(),
-            ..Default::default()
-        }
+        RepoEditOptionsBuilder(
+            RepoEditOptions {
+                name: name.into(),
+                ..Default::default()
+            }
+        )
     }
 
     pub fn description<D>(&mut self, description: D) -> &mut Self
     where
         D: Into<String>,
     {
-        self.description = Some(description.into());
+        self.0.description = Some(description.into());
         self
     }
 
     pub fn homepage<H>(&mut self, homepage: H) -> &mut Self
     where
         H: Into<String>,
     {
-        self.homepage = Some(homepage.into());
+        self.0.homepage = Some(homepage.into());
         self
     }
 
     pub fn private(&mut self, private: bool) -> &mut Self {
-        self.private = Some(private);
+        self.0.private = Some(private);
         self
     }
 
     pub fn has_issues(&mut self, has_issues: bool) -> &mut Self {
-        self.has_issues = Some(has_issues);
+        self.0.has_issues = Some(has_issues);
         self
     }
 
     pub fn has_projects(&mut self, has_projects: bool) -> &mut Self {
-        self.has_projects = Some(has_projects);
+        self.0.has_projects = Some(has_projects);
         self
     }
 
     pub fn has_wiki(&mut self, has_wiki: bool) -> &mut Self {
-        self.has_wiki = Some(has_wiki);
+        self.0.has_wiki = Some(has_wiki);
         self
     }
 
     pub fn default_branch<DB>(&mut self, default_branch: DB) -> &mut Self
     where
         DB: Into<String>,
     {
-        self.default_branch = Some(default_branch.into());
+        self.0.default_branch = Some(default_branch.into());
         self
     }
 
     pub fn allow_squash_merge(&mut self, allow_squash_merge: bool) -> &mut Self {
-        self.allow_squash_merge = Some(allow_squash_merge);
+        self.0.allow_squash_merge = Some(allow_squash_merge);
         self
     }
 
     pub fn allow_merge_commit(&mut self, allow_merge_commit: bool) -> &mut Self {
-        self.allow_merge_commit = Some(allow_merge_commit);
+        self.0.allow_merge_commit = Some(allow_merge_commit);
         self
     }
 
     pub fn allow_rebase_merge(&mut self, allow_rebase_merge: bool) -> &mut Self {
-        self.allow_rebase_merge = Some(allow_rebase_merge);
+        self.0.allow_rebase_merge = Some(allow_rebase_merge);
         self
     }
 
     pub fn build(&self) -> RepoEditOptions {
         RepoEditOptions::new(
-            self.name.as_str(),
-            self.description.clone(),
-            self.homepage.clone(),
-            self.private,
-            self.has_issues,
-            self.has_projects,
-            self.has_wiki,
-            self.default_branch.clone(),
-            self.allow_squash_merge,
-            self.allow_merge_commit,
-            self.allow_rebase_merge,
+            self.0.name.as_str(),
```

Comment:
oh interesting. I like this. I should probably be doing this pattern with the other builders.